# Bioinformatics and Genomics - References

## Academic Papers and Research

### Foundational Papers

1. **The Sequence of the Human Genome**
   - https://www.science.org/doi/10.1126/science.1058040
   - Landmark paper on the Human Genome Project

2. **Initial sequencing and analysis of the human genome**
   - https://www.nature.com/articles/35057062
   - Nature publication of the human genome sequence

3. **A global reference for human genetic variation**
   - https://www.nature.com/articles/nature15393
   - 1000 Genomes Project Phase 3

4. **The ENCODE (Encyclopedia of DNA Elements) Project**
   - https://www.nature.com/articles/nature11247
   - Comprehensive catalog of functional elements

5. **Standards and guidelines for the interpretation of sequence variants**
   - https://www.nature.com/articles/gim201530
   - ACMG/AMP variant interpretation guidelines

6. **Highly accurate protein structure prediction with AlphaFold**
   - https://www.nature.com/articles/s41586-021-03819-2
   - Revolutionary AI-based structure prediction

### Methods Papers

7. **Fast and accurate short read alignment with Burrows-Wheeler transform**
   - https://academic.oup.com/bioinformatics/article/25/14/1754/225615
   - BWA algorithm paper

8. **STAR: ultrafast universal RNA-seq aligner**
   - https://academic.oup.com/bioinformatics/article/29/1/15/272537
   - STAR aligner for RNA-seq

9. **The Genome Analysis Toolkit: A MapReduce framework for analyzing NGS data**
   - https://genome.cshlp.org/content/20/9/1297
   - GATK framework paper

10. **Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2**
    - https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8
    - DESeq2 methodology

11. **limma powers differential expression analyses for RNA-sequencing and microarray studies**
    - https://academic.oup.com/nar/article/43/7/e47/2414268
    - limma-voom methodology

12. **Salmon provides fast and bias-aware quantification of transcript expression**
    - https://www.nature.com/articles/nmeth.4197
    - Salmon quantification method

13. **A scaling normalization method for differential expression analysis of RNA-seq data**
    - https://genomebiology.biomedcentral.com/articles/10.1186/gb-2010-11-3-r25
    - TMM normalization method

14. **Integrative clustering of multiple genomic data types**
    - https://www.nature.com/articles/nmeth.1344
    - Multi-omics integration methods

15. **MaxQuant enables high peptide identification rates**
    - https://www.nature.com/articles/nbt.1511
    - MaxQuant proteomics platform

## Databases and Resources

### Genomic Databases

16. **NCBI (National Center for Biotechnology Information)**
    - https://www.ncbi.nlm.nih.gov/
    - Comprehensive biological database resource

17. **Ensembl Genome Browser**
    - https://www.ensembl.org/
    - Genome annotation and analysis

18. **UCSC Genome Browser**
    - https://genome.ucsc.edu/
    - Interactive genome visualization

19. **gnomAD (Genome Aggregation Database)**
    - https://gnomad.broadinstitute.org/
    - Population genetic variation database

20. **ClinVar**
    - https://www.ncbi.nlm.nih.gov/clinvar/
    - Clinical variant database

21. **dbSNP**
    - https://www.ncbi.nlm.nih.gov/snp/
    - Single nucleotide polymorphism database

22. **dbGaP (Database of Genotypes and Phenotypes)**
    - https://www.ncbi.nlm.nih.gov/gap/
    - Genotype-phenotype studies

23. **GEO (Gene Expression Omnibus)**
    - https://www.ncbi.nlm.nih.gov/geo/
    - Gene expression and functional genomics data

24. **SRA (Sequence Read Archive)**
    - https://www.ncbi.nlm.nih.gov/sra
    - Raw sequencing data repository

25. **European Nucleotide Archive (ENA)**
    - https://www.ebi.ac.uk/ena/browser/
    - European sequence data archive

### Protein Databases

26. **UniProt**
    - https://www.uniprot.org/
    - Comprehensive protein knowledgebase

27. **Protein Data Bank (PDB)**
    - https://www.rcsb.org/
    - 3D protein structure database

28. **SWISS-MODEL**
    - https://swissmodel.expasy.org/
    - Protein structure homology modeling

29. **AlphaFold Protein Structure Database**
    - https://alphafold.ebi.ac.uk/
    - AI-predicted protein structures

30. **STRING**
    - https://string-db.org/
    - Protein-protein interaction networks

31. **IntAct**
    - https://www.ebi.ac.uk/intact/
    - Molecular interaction database

32. **InterPro**
    - https://www.ebi.ac.uk/interpro/
    - Protein families and domains

33. **Pfam**
    - https://www.ebi.ac.uk/interpro/entry/pfam/
    - Protein family database

34. **PRIDE (PRoteomics IDEntifications)**
    - https://www.ebi.ac.uk/pride/
    - Proteomics data repository

### Pathway and Ontology Databases

35. **Gene Ontology (GO)**
    - http://geneontology.org/
    - Structured vocabulary for gene function

36. **KEGG (Kyoto Encyclopedia of Genes and Genomes)**
    - https://www.genome.jp/kegg/
    - Pathway and disease database

37. **Reactome**
    - https://reactome.org/
    - Biological pathway database

38. **WikiPathways**
    - https://www.wikipathways.org/
    - Community-curated pathways

39. **Human Phenotype Ontology (HPO)**
    - https://hpo.jax.org/
    - Phenotypic abnormalities ontology

40. **Disease Ontology**
    - https://disease-ontology.org/
    - Disease classification ontology

### Clinical and Disease Databases

41. **COSMIC (Catalogue of Somatic Mutations in Cancer)**
    - https://cancer.sanger.ac.uk/cosmic
    - Cancer mutation database

42. **OMIM (Online Mendelian Inheritance in Man)**
    - https://www.omim.org/
    - Human genes and genetic disorders

43. **ClinGen**
    - https://clinicalgenome.org/
    - Clinical genome resource

44. **PharmGKB**
    - https://www.pharmgkb.org/
    - Pharmacogenomics knowledge base

45. **The Cancer Genome Atlas (TCGA)**
    - https://www.cancer.gov/tcga
    - Cancer genomics data

46. **ICGC (International Cancer Genome Consortium)**
    - https://dcc.icgc.org/
    - International cancer genomics data

47. **GTEx (Genotype-Tissue Expression)**
    - https://gtexportal.org/
    - Tissue-specific gene expression

48. **UK Biobank**
    - https://www.ukbiobank.ac.uk/
    - Large-scale biomedical database

## Tools and Software

### Sequence Alignment and Mapping

49. **BWA (Burrows-Wheeler Aligner)**
    - https://github.com/lh3/bwa
    - DNA sequence aligner

50. **BWA-MEM2**
    - https://github.com/bwa-mem2/bwa-mem2
    - Faster BWA-MEM implementation

51. **Bowtie2**
    - http://bowtie-bio.sourceforge.net/bowtie2/
    - Fast and sensitive read aligner

52. **STAR**
    - https://github.com/alexdobin/STAR
    - RNA-seq aligner

53. **HISAT2**
    - http://daehwankimlab.github.io/hisat2/
    - Graph-based alignment

54. **Minimap2**
    - https://github.com/lh3/minimap2
    - Long-read and assembly alignment

### Variant Calling and Analysis

55. **GATK (Genome Analysis Toolkit)**
    - https://gatk.broadinstitute.org/
    - Variant discovery toolkit

56. **DeepVariant**
    - https://github.com/google/deepvariant
    - Deep learning variant caller

57. **FreeBayes**
    - https://github.com/freebayes/freebayes
    - Bayesian genetic variant detector

58. **Strelka2**
    - https://github.com/Illumina/strelka
    - Germline and somatic variant caller

59. **Mutect2**
    - https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2
    - Somatic variant caller

60. **VarScan**
    - http://varscan.sourceforge.net/
    - Variant detection tool

61. **bcftools**
    - http://samtools.github.io/bcftools/
    - VCF/BCF manipulation

62. **VEP (Variant Effect Predictor)**
    - https://www.ensembl.org/vep
    - Variant annotation tool

63. **ANNOVAR**
    - https://annovar.openbioinformatics.org/
    - Functional variant annotation

64. **SnpEff**
    - http://pcingola.github.io/SnpEff/
    - Genetic variant annotation

### Structural Variant Analysis

65. **Manta**
    - https://github.com/Illumina/manta
    - Structural variant caller

66. **DELLY**
    - https://github.com/dellytools/delly
    - Structural variant discovery

67. **LUMPY**
    - https://github.com/arq5x/lumpy-sv
    - General SV detection

68. **CNVkit**
    - https://cnvkit.readthedocs.io/
    - Copy number analysis

69. **GRIDSS**
    - https://github.com/PapenfussLab/gridss
    - Genome rearrangement identification

### RNA-seq Analysis

70. **DESeq2**
    - https://bioconductor.org/packages/DESeq2
    - Differential expression analysis

71. **edgeR**
    - https://bioconductor.org/packages/edgeR
    - Differential expression analysis

72. **Salmon**
    - https://combine-lab.github.io/salmon/
    - Transcript quantification

73. **kallisto**
    - https://pachterlab.github.io/kallisto/
    - Quantification of RNA-seq

74. **RSEM**
    - https://deweylab.github.io/RSEM/
    - RNA-seq quantification

75. **StringTie**
    - https://ccb.jhu.edu/software/stringtie/
    - Transcript assembly

76. **featureCounts**
    - http://subread.sourceforge.net/
    - Read counting

77. **HTSeq**
    - https://htseq.readthedocs.io/
    - High-throughput sequencing analysis

### Single-Cell Analysis

78. **Seurat**
    - https://satijalab.org/seurat/
    - R toolkit for single-cell genomics

79. **Scanpy**
    - https://scanpy.readthedocs.io/
    - Python single-cell analysis

80. **Cell Ranger**
    - https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger
    - 10x Genomics analysis

81. **Monocle**
    - https://cole-trapnell-lab.github.io/monocle3/
    - Single-cell trajectory analysis

82. **scVI-tools**
    - https://scvi-tools.org/
    - Deep learning for single-cell

83. **CellTypist**
    - https://www.celltypist.org/
    - Automated cell type annotation

84. **SCENIC**
    - https://scenic.aertslab.org/
    - Gene regulatory network inference

### Metagenomics

85. **QIIME2**
    - https://qiime2.org/
    - Microbiome bioinformatics platform

86. **MetaPhlAn**
    - https://huttenhower.sph.harvard.edu/metaphlan/
    - Metagenomic taxonomic profiling

87. **Kraken2**
    - https://ccb.jhu.edu/software/kraken2/
    - Taxonomic classification

88. **HUMAnN**
    - https://huttenhower.sph.harvard.edu/humann/
    - Functional profiling

89. **mothur**
    - https://mothur.org/
    - Microbial community analysis

90. **MEGAHIT**
    - https://github.com/voutcn/megahit
    - Metagenome assembly

91. **metaSPAdes**
    - https://cab.spbu.ru/software/spades/
    - Metagenomic assembler

92. **CheckM**
    - https://ecogenomics.github.io/CheckM/
    - Assess genome quality

### Proteomics

93. **MaxQuant**
    - https://www.maxquant.org/
    - Quantitative proteomics

94. **Proteome Discoverer**
    - https://www.thermofisher.com/proteome-discoverer
    - MS data analysis platform

95. **Perseus**
    - https://maxquant.net/perseus/
    - Proteomics data analysis

96. **Skyline**
    - https://skyline.ms/
    - Targeted proteomics

97. **MSFragger**
    - https://msfragger.nesvilab.org/
    - Ultra-fast database searching

98. **SearchGUI**
    - http://compomics.github.io/projects/searchgui
    - Search engine interface

99. **PeptideShaker**
    - http://compomics.github.io/projects/peptide-shaker
    - Peptide identification

### Structural Biology

100. **AlphaFold**
     - https://github.com/google-deepmind/alphafold
     - AI protein structure prediction

101. **RoseTTAFold**
     - https://www.bakerlab.org/research/rosettafold/
     - Deep learning structure prediction

102. **GROMACS**
     - https://www.gromacs.org/
     - Molecular dynamics

103. **AMBER**
     - https://ambermd.org/
     - Molecular dynamics suite

104. **AutoDock**
     - http://autodock.scripps.edu/
     - Molecular docking

105. **GOLD**
     - https://www.ccdc.cam.ac.uk/solutions/software/gold/
     - Protein-ligand docking

106. **PyMOL**
     - https://pymol.org/
     - Molecular visualization

107. **ChimeraX**
     - https://www.cgl.ucsf.edu/chimerax/
     - Structure visualization

108. **Modeller**
     - https://salilab.org/modeller/
     - Homology modeling

### Workflow Management

109. **Snakemake**
     - https://snakemake.readthedocs.io/
     - Workflow management system

110. **Nextflow**
     - https://www.nextflow.io/
     - Data-driven computational pipelines

111. **WDL (Workflow Description Language)**
     - https://openwdl.org/
     - Workflow specification language

112. **Cromwell**
     - https://cromwell.readthedocs.io/
     - WDL execution engine

113. **nf-core**
     - https://nf-co.re/
     - Community Nextflow pipelines

114. **Galaxy**
     - https://galaxyproject.org/
     - Web-based analysis platform

### Quality Control

115. **FastQC**
     - https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
     - Sequence quality control

116. **MultiQC**
     - https://multiqc.info/
     - Aggregate QC reports

117. **fastp**
     - https://github.com/OpenGene/fastp
     - Fast FASTQ preprocessing

118. **Trimmomatic**
     - http://www.usadellab.org/cms/?page=trimmomatic
     - Read trimming

119. **Cutadapt**
     - https://cutadapt.readthedocs.io/
     - Adapter removal

120. **Picard**
     - https://broadinstitute.github.io/picard/
     - BAM/VCF utilities

### Visualization

121. **IGV (Integrative Genomics Viewer)**
     - https://igv.org/
     - Interactive genome visualization

122. **JBrowse**
     - https://jbrowse.org/
     - Genome browser

123. **Circos**
     - http://circos.ca/
     - Circular genome visualization

124. **EnhancedVolcano**
     - https://bioconductor.org/packages/EnhancedVolcano
     - Volcano plot visualization

125. **ComplexHeatmap**
     - https://bioconductor.org/packages/ComplexHeatmap
     - Heatmap visualization

## Standards and Guidelines

### Data Standards

126. **GA4GH (Global Alliance for Genomics and Health)**
     - https://www.ga4gh.org/
     - Genomic data sharing framework

127. **FAIR Principles**
     - https://www.go-fair.org/fair-principles/
     - Data management principles

128. **SAM/BAM Format Specification**
     - https://samtools.github.io/hts-specs/
     - Sequence alignment format

129. **VCF Specification**
     - https://samtools.github.io/hts-specs/VCFv4.3.pdf
     - Variant call format

130. **CRAM Format**
     - https://www.ebi.ac.uk/ena/cram
     - Compressed sequence format

131. **GFF3 Specification**
     - https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md
     - General feature format

132. **MAGE-TAB**
     - https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html
     - Microarray data format

### Clinical Guidelines

133. **ACMG Standards and Guidelines**
     - https://www.acmg.net/ACMG/Medical-Genetics-Practice-Resources/Practice-Guidelines.aspx
     - Clinical genetics guidelines

134. **AMP/CAP Cancer Variant Interpretation Guidelines**
     - https://www.amp.org/resources/
     - Somatic variant interpretation

135. **ClinGen Sequence Variant Interpretation**
     - https://clinicalgenome.org/working-groups/sequence-variant-interpretation/
     - Variant curation guidelines

136. **NCCN Clinical Practice Guidelines**
     - https://www.nccn.org/guidelines/
     - Cancer treatment guidelines

### Best Practices

137. **ENCODE Pipelines**
     - https://www.encodeproject.org/pipelines/
     - Analysis best practices

138. **GATK Best Practices**
     - https://gatk.broadinstitute.org/hc/en-us/sections/360007226651-Best-Practices-Workflows
     - Variant calling workflows

139. **BLUEPRINT Analysis Guidelines**
     - https://www.blueprint-epigenome.eu/
     - Epigenome analysis

140. **Human Cell Atlas Analysis**
     - https://www.humancellatlas.org/
     - Single-cell analysis standards

## Books and Learning Resources

### Textbooks

141. **"Bioinformatics: Sequence and Genome Analysis" by David Mount**
     - https://www.cshlpress.com/default.tpl?cart=&action=full&eession=&--eession=&neession=1423595155&key=A17950&template=SAMPLEchapter
     - Comprehensive bioinformatics textbook

142. **"Biological Sequence Analysis" by Durbin et al.**
     - https://www.cambridge.org/core/books/biological-sequence-analysis/920BB6D1B8B60FB92D9E6981E93BD669
     - Probabilistic models in bioinformatics

143. **"Bioinformatics Data Skills" by Vince Buffalo**
     - https://www.oreilly.com/library/view/bioinformatics-data-skills/9781449367480/
     - Reproducible research practices

144. **"Computational Genome Analysis" by Deonier et al.**
     - https://link.springer.com/book/10.1007/b137547
     - Genome analysis methods

145. **"Proteomics for Biological Discovery" by Veenstra and Yates**
     - https://onlinelibrary.wiley.com/doi/book/10.1002/0471721646
     - Proteomics methods

146. **"RNA-seq Data Analysis" by Han et al.**
     - https://www.routledge.com/RNA-seq-Data-Analysis-A-Practical-Approach/Han-Kong/p/book/9781466595002
     - Practical RNA-seq guide

147. **"Single-Cell Best Practices" by Luecken et al.**
     - https://www.sc-best-practices.org/
     - Single-cell analysis guide

148. **"An Introduction to Systems Biology" by Uri Alon**
     - https://www.routledge.com/An-Introduction-to-Systems-Biology-Design-Principles-of-Biological-Circuits/Alon/p/book/9781439837177
     - Systems biology principles

### Online Courses

149. **EMBL-EBI Training**
     - https://www.ebi.ac.uk/training/
     - Bioinformatics courses

150. **Coursera - Genomic Data Science Specialization**
     - https://www.coursera.org/specializations/genomic-data-science
     - Johns Hopkins genomics courses

151. **edX - Principles of Biochemistry**
     - https://www.edx.org/learn/biochemistry
     - Biochemistry foundations

152. **Bioconductor Training**
     - https://bioconductor.org/help/education-training/
     - R/Bioconductor courses

153. **Harvard Chan Bioinformatics Core Training**
     - https://hbctraining.github.io/main/
     - Hands-on workshops

154. **Rosalind - Bioinformatics Learning**
     - https://rosalind.info/
     - Problem-based learning

155. **MIT OpenCourseWare - Computational Biology**
     - https://ocw.mit.edu/courses/6-047-computational-biology-fall-2015/
     - University course materials

156. **Cold Spring Harbor Laboratory Courses**
     - https://meetings.cshl.edu/courseshome.aspx
     - Intensive bioinformatics courses

## Community Resources

### Organizations

157. **ISCB (International Society for Computational Biology)**
     - https://www.iscb.org/
     - Professional organization

158. **ASHG (American Society of Human Genetics)**
     - https://www.ashg.org/
     - Human genetics society

159. **AACR (American Association for Cancer Research)**
     - https://www.aacr.org/
     - Cancer research organization

160. **HUPO (Human Proteome Organization)**
     - https://www.hupo.org/
     - Proteomics organization

### Conferences

161. **ISMB (Intelligent Systems for Molecular Biology)**
     - https://www.iscb.org/about-ismb
     - Premier bioinformatics conference

162. **RECOMB (Research in Computational Molecular Biology)**
     - https://recomb.org/
     - Computational biology conference

163. **ASHG Annual Meeting**
     - https://www.ashg.org/meetings/
     - Human genetics conference

164. **Genome Informatics**
     - https://meetings.cshl.edu/meetings.aspx?meet=INFO
     - Cold Spring Harbor meeting

165. **AGBT (Advances in Genome Biology and Technology)**
     - https://www.agbt.org/
     - Genomic technologies conference

### Forums and Communities

166. **Biostars**
     - https://www.biostars.org/
     - Bioinformatics Q&A

167. **SEQanswers**
     - http://seqanswers.com/
     - Next-gen sequencing forum

168. **Bioinformatics Stack Exchange**
     - https://bioinformatics.stackexchange.com/
     - Q&A community

169. **Reddit r/bioinformatics**
     - https://www.reddit.com/r/bioinformatics/
     - Community discussions

170. **Bioconductor Support**
     - https://support.bioconductor.org/
     - R/Bioconductor help

### Newsletters and Blogs

171. **Nature Methods**
     - https://www.nature.com/nmeth/
     - Methods development journal

172. **Genome Biology**
     - https://genomebiology.biomedcentral.com/
     - Open access genomics journal

173. **Bioinformatics Journal**
     - https://academic.oup.com/bioinformatics
     - Applications note journal

174. **PLOS Computational Biology**
     - https://journals.plos.org/ploscompbiol/
     - Open access journal

175. **Getting Genetics Done Blog**
     - https://www.gettinggeneticsdone.com/
     - Practical bioinformatics tips

## Cloud and Computing Platforms

### Cloud Genomics Platforms

176. **Terra (Broad Institute)**
     - https://terra.bio/
     - Cloud-native genomics platform

177. **DNAnexus**
     - https://www.dnanexus.com/
     - Enterprise genomics platform

178. **Seven Bridges**
     - https://www.sevenbridges.com/
     - Genomics cloud platform

179. **Google Cloud Life Sciences**
     - https://cloud.google.com/life-sciences
     - GCP genomics services

180. **AWS Genomics**
     - https://aws.amazon.com/health/genomics/
     - AWS genomics solutions

181. **Microsoft Genomics**
     - https://azure.microsoft.com/en-us/services/genomics/
     - Azure genomics service

182. **Illumina BaseSpace**
     - https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub.html
     - Illumina cloud platform

### High-Performance Computing

183. **SLURM**
     - https://slurm.schedmd.com/
     - Cluster workload manager

184. **LSF (Load Sharing Facility)**
     - https://www.ibm.com/docs/en/spectrum-lsf
     - Enterprise scheduler

185. **PBS/Torque**
     - https://www.openpbs.org/
     - Resource manager

186. **Singularity/Apptainer**
     - https://apptainer.org/
     - HPC containerization

187. **Bioconda**
     - https://bioconda.github.io/
     - Bioinformatics software distribution

188. **Biocontainers**
     - https://biocontainers.pro/
     - Container registry

## Reproducibility Tools

189. **DVC (Data Version Control)**
     - https://dvc.org/
     - Data and model versioning

190. **Git-LFS**
     - https://git-lfs.github.com/
     - Large file storage

191. **Zenodo**
     - https://zenodo.org/
     - Research data repository

192. **Figshare**
     - https://figshare.com/
     - Research output sharing

193. **Protocols.io**
     - https://www.protocols.io/
     - Protocol sharing platform

194. **Jupyter Notebooks**
     - https://jupyter.org/
     - Interactive computing

195. **R Markdown**
     - https://rmarkdown.rstudio.com/
     - Reproducible documents

## Emerging Technologies

196. **Oxford Nanopore Technologies**
     - https://nanoporetech.com/
     - Long-read sequencing

197. **PacBio**
     - https://www.pacb.com/
     - SMRT sequencing

198. **10x Genomics**
     - https://www.10xgenomics.com/
     - Single-cell and spatial genomics

199. **Spatial Transcriptomics (Visium)**
     - https://www.10xgenomics.com/products/spatial-gene-expression
     - Spatial gene expression

200. **CRISPR Tools (Benchling)**
     - https://www.benchling.com/
     - CRISPR design and analysis
